common/src/main/java/com/sequenceiq/cloudbreak/common/converter/MissingResourceNameGenerator.java

```diff
@@ -1,4 +1,4 @@
-package com.sequenceiq.cloudbreak.service;
+package com.sequenceiq.cloudbreak.common.converter;
```

Comment:
I suggest doing a separate PR for refactoring the missing name generator code, since it is very cross cutting and not as focused on redbeams.

Reply:
Yes, would be nice - on the other hand doing a separate PR slows down development a lot (running checks on github, waiting and waiting for reviews), during which I cannot really make any progress. So for smaller refactors I do not mind not doing a separate PR, so as not to be blocked.